.eslintrc.json

```diff
@@ -5,13 +5,29 @@
     "standard",
     "prettier",
     "prettier/standard",
+    "plugin:unicorn/recommended",
     "plugin:ava/recommended",
     "plugin:you-dont-need-lodash-underscore/all"
   ],
   "rules": {
     "no-process-exit": 0,
     "object-shorthand": "error",
     "require-await": "error",
+    "unicorn/custom-error-definition": 2,
+    "unicorn/no-unused-properties": 2,
+    "unicorn/filename-case": [2, { "case": "kebabCase", "ignore": [".*.md"] }],
+    "unicorn/explicit-length-check": [2, { "non-zero": "not-equal" }],
+    "unicorn/no-process-exit": 0,
+    "unicorn/no-null": 0,
+    "unicorn/prevent-abbreviations": 0,
+    "unicorn/number-literal-case": 0,
+    "unicorn/expiring-todo-comments": 0,
+    "unicorn/no-fn-reference-in-iterator": 0,
```

Comment:
I am wondering what you think about this one. [See rule documentation](https://github.com/sindresorhus/eslint-plugin-unicorn/blob/master/docs/rules/no-fn-reference-in-iterator.md).
I left it out here mostly because it would require too much refactoring.

Reply:
Nice, this can prevent some bugs